api/gitlab.go

```diff
@@ -19,7 +19,7 @@ type GitLabGateway struct {
 }
 
 var gitlabPathRegexp = regexp.MustCompile("^/gitlab/?")
-var gitlabAllowedRegexp = regexp.MustCompile("^/gitlab/repository/(files|commits|tree)/?")
+var gitlabAllowedRegexp = regexp.MustCompile("^/gitlab/merge_requests|(repository/(files|commits|tree|compare|branches))/?")
```

Comment:
```suggestion
var gitlabAllowedRegexp = regexp.MustCompile("^/gitlab/(merge_requests|(repository/(files|commits|tree|compare|branches)))/?")
```
this is safer.

I don't like having a regex here at all, but we are not going to refactor this now i think.